### PR TITLE
ENH: Add DenseNet IsoIntensity infant MRI brain segmentation review.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add DenseNet IsoIntensity infant MRI brain segmentation review.

Add new tags.

### CODE REVIEW

File: deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+layout: review
+title: "Exclusive Independent Probability Estimation using Deep 3D Fully
+Convolutional DenseNets for IsoIntense Infant Brain MRI Segmentation"
+tags: deep-learning segmentation brain DenseNets
```
Comment (jodoin) [inline deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md]: DenseNets is kind of too specific for a tag.  I would use the usual CNN.

File: deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- Average Surface Distance (ASD)
+
+## Data and Benchmarking
+- [2017 MICCAI iSeg challenge dataset](http://iseg2017.web.unc.edu/download/)
+- T1- and T2-weighed MRIs of 10 subjects
+  - Data augmentation: use patches on input images; rotated patches
+  - Prediction fusion: soft voting approach
+
+# Results
+- Best results with Exclusive multi-label multi-class training
```
Comment (jodoin) [inline deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md]: exclusive

File: deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md (hunk 1 above)
Comment (jhlegarreta) [reply deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md]: Done in 578fb93. Thanks for the review.

File: deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md (hunk 2 above)
Comment (jhlegarreta) [reply deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md]: Done in 578fb93. Thanks for the review.

**Pull request merged**